backend/ml/management/commands/ml_train.py

```diff
@@ -50,48 +50,52 @@
 """
 
 
-def fetch_data():
+def fetch_data(trusted=True):
```

Comment:
Maybe a parameter such as `trusted_only=True` would be more explicit.
(`trusted=False` also includes comparisons from trusted users)

Reply:
Yes great idea thanks